src/pages/components/utilities.mdx

```diff
@@ -0,0 +1,121 @@
+---
+title: Utilities
+description: >
+  Utility functions enable advanced theme usage. They let you customize
+  components in a granular way.
+package: '@zendeskgarden/react-theming'
+components:
+  - theming/src/utils/arrowStyles.ts
+  - theming/src/utils/getColor.ts
+  - theming/src/utils/getLineHeight.ts
+  - theming/src/utils/mediaQuery.ts
+  - theming/src/utils/menuStyles.ts
+  - theming/src/utils/retrieveComponentStyles.ts
+  - theming/src/utils/withTheme.ts
+---
+
+## How to use it
+
+### Menu and arrow styles
+
+Use menu and arrow styling utilities together to ease the production of
```

Comment:
What does "ease the production" mean? Do we mean: Use menu and arrow styling utilities to make it easier to build Garden-styled menu components?

Reply:
Yes. In fact, we can probably simplify to:

> Use menu and arrow styling utilities to build Garden-styled menu components.